guides/hosting/configurations/observability/profiling.md

```diff
@@ -0,0 +1,76 @@
+---
+nav:
+  title: Profiling / Tracing
+  position: 20
+
+---
+
+# Profiling
+
+Shopware provides a built-in profiler abstraction to measure the performance of code parts and publish this data to a profiler backend.
+
+## Enabling the profiler backends
+
+By default only the Stopwatch profiler (Symfony Profiler Bar) is enabled. To enable the other profiler backends, you have to add the following configuration to your `config/packages/shopware.yaml` file:
+
+```yaml
+shopware:
+    profiler:
+        integrations:
+            - Symfony
+            - Datadog
+            - Tideways
+            - OpenTelemetry
+```
+
+::: info
+The OpenTelemetry profiler is not installed by default. Checkout the [OpenTelemetry Integration](./opentelemetry.md) to learn how to install it. 
```

Comment:
DD and tideways also need to have extensions installed or? Does it make sense to add a little section for each of these 

E.g. link to dd agent installation guide or tideways extension and so on